### ISSUE TITLE
SR1 - Services by Category missing underline on hover (when description is missing)

### ISSUE THREAD

**Issue by mmafe** [brharp/hjckrrh#225]
![sr2-no-underline-hover](https://cloud.githubusercontent.com/assets/1927902/10052390/a3fd1c82-61f4-11e5-93b9-ae014e78aa71.png)

Appears to be caused by a margin-bottom: 0 style overridding the media-heading style.

Fix in media styles.

**Comment by jpinsent** [comment]
This looks like it's been fixed along with issue #241. All links in the Services by Category page now have an underline on hover.
